src/ghci/stderr.rs

```diff
@@ -19,17 +22,21 @@ pub enum StderrEvent {
     GetBuffer { sender: oneshot::Sender<String> },
 }
 
-pub struct GhciStderr {
+pub struct GhciStderr<W> {
```

Comment:
If `W` is always `Box<dyn GhciWrite>`, we might just want to make that the type.

Reply:
Done in https://github.com/MercuryTechnologies/ghciwatch/pull/175/commits/388f83624873866f142dcb09d63be112daac9d58 and https://github.com/MercuryTechnologies/ghciwatch/pull/175/commits/8d0e44680084b74e6b01ea8a8a21ec8cc29fe081